src/App.css

```diff
@@ -158,8 +158,33 @@ label.hover {
 
 :root {
   --color-gray: #767676;
+  --color-light-green: #88c057;
+  --color-light-orange: rgba(238, 175, 75, 0.67);
+  --color-light-red: rgba(237, 63, 50, 0.67);
 }
 
+/* Layout */
+.flex {
+  display: flex;
+}
+
+.row {
```

Comment:
🔮 tip for your future projects:

Rows are one of the most common things to use when you build a webpage, think about being more specific in the naming, you technically could build a `row` with "anything". That said, I think that in a bigger project, it would make sense to rename this selector to `flex-row`, you can also have other rows like `table-row` or `grid-row`.